### DISCUSSION TITLE
How do I change/role a new ID?

### DISCUSSION THREAD

**Discussion by DaReaLDeviL** [rustdesk/rustdesk#3056] [Q&A]
I cloned a vm and had RustDesk (1.1.9) installed on it. So I need to change now the ID on the new clone. But my attemps to do so failed.

-tried the "change ID"
-uninstalled, reboot, installed new
-installed nightly build 1.2
-deleted files C:\Users\UserName\AppData\Roaming\RustDesk

is there something else to get a new ID?

**Comment by korneevigor** [comment]
Нужно остановить службу RustDesk Service и закрыть приложение RustDesk

Откройте командную строку (cmd) с правами администратора.

Выберите папку, введя команду:

cd C:\Windows\ServiceProfiles\LocalService\AppData\Roaming\RustDesk\config

Затем запустите блокнот с редактируемым файлом командой notepad Rustdesk.toml
4. Замените в файле Rustdesk.toml следующие значения:

id = '0'
password = 'ВАШ_ПАРОЛЬ'
salt = ''
Сохраните изменения и перезагрузите компьютер.
